content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,707 @@
+:guilabel:`Discard` button to revert to the original information.
+
+.. image:: referrals/levels.png
+   :align: center
+   :alt: A levels form in edit mode.
+
+.. warning::
+   It is not advised to edit the images. An image file must have a transparent background in order
+   for it to render properly. Only users with knowledge about transparent images should attempt
+   adjusting any images in the Referrals application.
+
+   Once an image is changed and the level is saved, it is **not possible** to revert to the original
+   image. To revert to the original image, the Referrals application must be *uninstalled then
+   reinstalled.*
+
+Level up
+--------
+
+Once enough points have been accumulated to level up, the circle around the user's photo is
+completely filled in with cyan, a large image stating `Level up!` appears above the photo, and the
+phrase `Click to level up!` appears beneath the user's photo and current level.
+
+Click on the photo to level up the user. The user's avatar changes to the current level, and the
+ring around the photo is updated to indicate the current amount of points.
+
+.. image:: referrals/level-up.png
+   :align: center
+   :alt: A 'Click to level up!' appears beneath the user's image, and a large 'Level up!' appears
+         above their image.
+
+.. note::
+   Once a user has reached the highest configured level, they will continue to accrue points that
+   can be redeemed for rewards, but they are no longer able to level up. The ring around their photo
+   remains solid cyan.
+
+Alerts
+======
+
+In the Referrals application, it is possible to post a message, also referred to as an *alert*, at
+the top of the dashboard to share important information with users. These alerts appear as a thin
+semi-transparent banner, with the word `New!` appearing on the far left. The text for the alert is
+in the center of the banner, and on the far right side is an :guilabel:`X`.
+
+Alerts appear on the main dashboard for the specified time configured on the individual alert. If a
+user does not wish to see a specific alert again, click the :guilabel:`X` in the far right side of
+the alert. This removes the alert from the dashboard and will not appear again, even when opening
+the Referrals application for the first time in a new session.
+
+.. image:: referrals/alerts.png
+   :align: center
+   :alt: Two alert banners appear above the user's photo.
+
+Create an alert
+---------------
+
+To add a new alert, navigate to the :menuselection:`Referrals application --> Configuration -->
+Alerts`.
+
+Click :guilabel:`Create` and a blank alert form loads. Enter the following information on the form:
+
+- :guilabel:`Date From`: The date the alert starts. On this date, the alert will be visible on the
+  dashboard.
+- :guilabel:`Date To`: The date the alert ends. After this date, the alert will be hidden from view.
+- :guilabel:`Company`: The current company populates this field by default. To modify the company
```

Comment:
Do all users see this alert? only users with this company assigned, who sees it?

Reply:
Good catch! If a compony is specified, it's company specific. if blank, it's for everyone- I added this clarification.